presto-thrift-connector/src/test/java/com/facebook/presto/connector/thrift/util/TestRetryDriver.java

```diff
@@ -75,7 +76,8 @@ void testNoRetry()
             throws Exception
     {
         SyncTaskImpl task = new SyncTaskImpl(SUCCESS);
-        Integer result = retry.run("test", task);
+        RetryStats retryStats = new RetryStats();
+        Integer result = retry.run("test", retryStats, task);
```

Comment:
Does this need to be boxed (can it be null)?

Reply:
Allowing null will require adding checks to skip updating the stats.  Maybe creating a unused object here is easier?